src/main/java/com/glitchcog/fontificator/emoji/loader/EmojiParser.java

```diff
@@ -224,24 +222,70 @@ private void parseTwitchBadges(TypedEmojiMap badgeMap, String jsonData) throws I
     {
         JsonElement jsonElement = new JsonParser().parse(jsonData);
 
+        Boolean isNewStyle = jsonElement.getAsJsonObject().has("badge_sets");
```

Comment:
Not sure if the changes to this function are handling all possible error states.